packages/strapi-admin/admin/src/translations/en.json

```diff
@@ -1,7 +1,7 @@
 {
   "Analytics": "Analytics",
   "Content Manager": "Content Manager",
-  "Content Type Builder": "Content Type Builder",
+  "Content Type Builder": "Types Builder",
```

Comment:
We decided to keep the current name "Content-Types Builder". A Content-Type is a Collection or a Single Types, both together they are the Content-Types.